website/react-magma-docs/src/components/Masthead/index.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import { Link } from 'gatsby';
-import { Logo } from '../Logo';
+import { Logo, SmallLogo } from '../Logo';
+import { ButtonVariant, IconButton } from 'react-magma-dom';
 import styled from '@emotion/styled';
 import { Header } from '@cengage-patterns/header';
-import { magma, useMediaQuery, AppBarPosition, Spacer } from 'react-magma-dom';
+import { magma, AppBarPosition } from 'react-magma-dom';
```

Comment:
line 4 & 7 can go together.

Reply:
addressed in [ff537d3](https://github.com/cengage/react-magma/pull/645/commits/ff537d347b7a929a91636d72b3fddc05696d520e)